MAINTAINERS.md

```diff
@@ -0,0 +1,7 @@
+## Current maintainers of the project
+
+| Maintainer             | GitHub ID                                               |
+| ---------------------- | ------------------------------------------------------- |
+| George Jahad           | [GeorgeJahad](https://github.com/GeorgeJahad)           |
```

Comment:
Hi @demarillacizere I'm George Jahad, the G-Research software engineer listed above.  I know this PR is closed, but I do not want my name listed here.  Please push another branch that removes my name.

Thank you!